### PR TITLE
phemex fetchMarkets v2

### PR DESCRIPTION
**Pull request by ndubel**
https://github.com/ccxt/ccxt/issues/20230
I still use https://api.phemex.com/v1/exchange/public/products to get missing fields for some perpetual contracts

### CODE REVIEW

**Comment by carlosmiei** [comment]
Hello @ndubel, thanks will check it now.

File: ts/src/phemex.ts [hunk 1]
```diff
@@ -159,6 +159,7 @@ export default class phemex extends Exchange {
                 },
                 'v2': {
                     'get': {
+                        'public/products': 5,
```
Comment (carlosmiei) [inline ts/src/phemex.ts:162]: I'm afraid we lost some markets here, previously we had 808 and now 781
image

Comment (carlosmiei) [reply ts/src/phemex.ts:162]: On the other hand, we have some of the missing markets as well (for instance BTC/USDC was not available previously)
image

Comment (ndubel) [reply ts/src/phemex.ts:162]: that's what I told in https://github.com/ccxt/ccxt/issues/20230.
You can check, that on web https://phemex.com/spot/trade/ there is no SWEAT/USDT, XTP/USDT, INS/USDC and some other pairs (at least spot), also there is no any orderbook for this pairs

Comment (carlosmiei) [reply ts/src/phemex.ts:162]: @ndubel Yeah you;'re right, these are the ones missing but I can't find them in the platform
image

**Pull request merged**